lib/errors.js

```diff
@@ -83,11 +114,16 @@ class ProcessError extends CliError {
             output += chalk.yellow(`Exit code: ${this.options.code}\n\n`);
         }
 
-        if (verbose && this.options.stdout) {
```

Comment:
When testing different error cases, I stumbled over a situation where `stderr` was filled, but `stdout` was empty. I changed the logic here, so it doesn't depend on `stdout` only to log our error.